tests/unittests/tests-uri_parser/tests-uri_parser.c

```diff
@@ -134,6 +140,7 @@ static const validate_t validate_uris[] = {
         "2001:db8::1",
         "",
         "5own",
+        5,
         "/v=1",
         "",
         0),
```

Comment:
The `5own` tests fail now, due to "uri_parser_process_string()" now returning -1. Not sure if that is wanted, but if it is, you need to change this to 
```suggestion
        5,
        "/v=1",
        "",
        -1),
```

But as this might skip some valuable coverage tests, I'd rather go with creating another dedicated test vector with `5own` in the URI and replacing `5own` with a valid port in the others (or fix the function if it was not intended to fail if the port is not a valid int.